apps/src/templates/sectionsRefresh/AdvancedSettingToggles.jsx

```diff
@@ -34,35 +37,83 @@ export default function AdvancedSettingToggles({
 
   return (
     <div>
-      <ToggleSwitch
-        id={'uitest-pair-toggle'}
-        isToggledOn={section.pairingAllowed}
-        onToggle={e => {
-          handlePairProgrammingToggle(e);
-        }}
-        label={i18n.pairProgramming()}
-      />
-      <ToggleSwitch
-        id={'uitest-lock-toggle'}
-        isToggledOn={section.restrictSection}
-        onToggle={e => handleLockSectionToggle(e)}
-        label={i18n.restrictSectionAccess()}
-      />
-      {assignedUnitTextToSpeechEnabled && (
+      <div className={style.toolTipContainer}>
         <ToggleSwitch
-          id={'uitest-tts-toggle'}
-          isToggledOn={section.ttsAutoplayEnabled}
-          onToggle={e => handleTtsAutoplayEnabledToggle(e)}
-          label={i18n.enableTtsAutoplay()}
+          id={'uitest-pair-toggle'}
+          isToggledOn={section.pairingAllowed}
+          onToggle={e => {
+            handlePairProgrammingToggle(e);
+          }}
+          label={i18n.pairProgramming()}
         />
-      )}
-      {assignedUnitLessonExtrasAvailable && (
+        <span data-tip data-for={'pair-toggle-info'}>
```

Comment:
We do have a [HelpTip](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/HelpTip.jsx) component that has a lot of this logic but it's not quite the same -- it looks like we're using a different FA icon. Curious if you think it's worth pulling out the code in this PR into a similar component for reusability?